blocks/init/src/Blocks/components/icon/icon-editor.scss

```diff
@@ -0,0 +1,17 @@
+.icon-option-row {
+	display: grid;
+	align-items: center;
+	grid-template-columns: 0.6rem 1fr;
+	grid-gap: 1rem;
+
+	i {
+		margin: 0;
+		padding: 0;
+
+		svg {
+			width: 1.2rem;
+			height: 1.2rem;
+			transform: translateY(2px);
```

Comment:
px rem mixup 🙂 

We need to choose one and stick with it.

Reply:
While we did agree on using (r)ems everywhere, some things should stay in pixels, usually things like border thicknesses, shadow radii and minor transforms like this one, where you wouldn't want the base font size to affect the layout.